wandb/apis/public.py

```diff
@@ -5492,8 +5497,26 @@ def name(self):
     def _set_configure_launch_project(self, func):
         self.configure_launch_project = func
 
+    def _get_code_artifact(self, artifact_string):
+        artifact_string, _, is_id = util.parse_artifact_string(artifact_string)
+        if is_id:
+            code_artifact = Artifact.from_id(artifact_string, self._api._client)
+        else:
+            code_artifact = self._api.artifact(name=artifact_string, type="code")
+        if code_artifact is None:
+            raise LaunchError("No code artifact found")
+        return code_artifact
+
+    def _configure_launch_project_notebook(self, launch_project):
+        new_fname = convert_jupyter_notebook_to_script(
```

Comment:
I'm getting a failure here after switching from vscode notebook to standard:
```
Traceback (most recent call last):
  File "/Users/ben/.pyenv/versions/dev-sdk/bin/wandb", line 8, in <module>
    sys.exit(cli())
  File "/Users/ben/.pyenv/versions/3.10.9/envs/dev-sdk/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/Users/ben/.pyenv/versions/3.10.9/envs/dev-sdk/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/Users/ben/.pyenv/versions/3.10.9/envs/dev-sdk/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/ben/.pyenv/versions/3.10.9/envs/dev-sdk/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/ben/.pyenv/versions/3.10.9/envs/dev-sdk/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/Users/ben/wandb/wandb/cli/cli.py", line 106, in wrapper
    return func(*args, **kwargs)
  File "/Users/ben/wandb/wandb/cli/cli.py", line 1254, in launch
    wandb_launch.run(
  File "/Users/ben/wandb/wandb/sdk/launch/launch.py", line 263, in run
    submitted_run_obj = _run(
  File "/Users/ben/wandb/wandb/sdk/launch/launch.py", line 169, in _run
    launch_project = fetch_and_validate_project(launch_project, api)
  File "/Users/ben/wandb/wandb/sdk/launch/_project_spec.py", line 483, in fetch_and_validate_project
    launch_project._fetch_job()
  File "/Users/ben/wandb/wandb/sdk/launch/_project_spec.py", line 267, in _fetch_job
    job.configure_launch_project(self)
  File "/Users/ben/wandb/wandb/apis/public.py", line 5563, in _configure_launch_project_artifact
    self._configure_launch_project_notebook(launch_project, True)
  File "/Users/ben/wandb/wandb/apis/public.py", line 5528, in _configure_launch_project_notebook
    new_fname = convert_jupyter_notebook_to_script(
  File "/Users/ben/wandb/wandb/sdk/launch/utils.py", line 527, in convert_jupyter_notebook_to_script
    with open(os.path.join(project_dir, fname)) as fh:
FileNotFoundError: [Errno 2] No such file or directory: '/var/folders/g7/17k87lv56vx5kw_w9lrrpqpc0000gn/T/tmpji9n4pif/_session_history.ipynb'
```

I have a code artifact with just a notebook called `test.py` in it. Here is the run that produces the code artifact and uses the job: https://wandb.ai/bcanfieldsherman/uncategorized/runs/4rdn7i0y/artifacts?workspace=user-bcanfieldsherman